wiki/Guides/fr.md

```diff
@@ -34,6 +34,7 @@ Vous trouverez ci-dessous une liste triée des guides créés par les membres de
 
```

Comment:
*J'ai pas accès pour suggérer directement sur ces lignes, je les met donc ici.*

Pour retirer la mention d'outdated (de la ligne 1 à la ligne 4, que tu pourra faire juste après avoir appliqué ce qui suit), il faut : 

- Changer l'ordre des listes pour qu'elle soit alphabétique
- Rajouter dans cette même liste `- [How to resolve BSS issues](BSS_Issues)` qui manque
- Rajouter en dessous de la ligne 33 `- [osu!mania modding guide](osu!mania_modding_guide)` qui manque
- Modifier ligne 33 la phrase `Le moddage de votre map` par `Le modding de votre beatmap`
- Traduire `Skinning guides and important threads` de la ligne 45

Après ça ce sera pas mal.

Reply:
j'ai prévus une autre pr pour corriger le outdated de "guides", car j'aime pas mélanger plusieurs choses différentes dans une seule pr

Reply:
Comme tu veut, mais a ta place je le ferais, car ça éviterais de faire pleins de PR pour le même article, mais si tu souhaite faire comme ça aucun problème

Reply:
>Walavouchey — Aujourd’hui à 19:31
>add the article in one pr, and update the links and other stuff in the other file in a separate pr